itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/cache/TestCachedStoreUpdateUsingEvents.java

```diff
@@ -82,9 +84,9 @@ public void setUp() throws Exception {
     HiveMetaStore.HMSHandler.createDefaultCatalog(rawStore, new Warehouse(conf));
   }
 
-  private Database createTestDb(String dbName, String dbOwner) {
+  private Database createTestDb(String dbName, String dbOwner) throws IOException {
     String dbDescription = dbName;
-    String dbLocation = "file:/tmp";
+    String dbLocation = Files.createTempDirectory(dbName).toString();
```

Comment:
Even the dbName can be same across multiple tests, which can get cleaned up. Would it be a good idea to use the current timestamp or generate a random number and use that for the temporary directory ?

Reply:
yeah, the Files.createTempDirectory creates a temp directory by appending the timestamp. DbName is just used as a prefix for the directory name.